### PR TITLE
feat: add loading while buy backer & adjust hexagon

### PR DESCRIPTION
**Pull request by Julius-745**
Add Toast Loading While Buy Backer

### CODE REVIEW

File: src/routes/home/BakeOurProject.tsx [hunk 1]
```diff
@@ -1,19 +1,42 @@
...
+        const loadingToast = toast({
+          title:  "Loading",
+          description: "Processing...",
+          status: "info" ,
+          duration:  null, // Null duration to keep the loading toast open
+          isClosable: false,
+        })
+        loadingToast
+
+        await buyPackage.exec(Number(pkg.id)).then(() => toast.close(loadingToast))
```
Comment (vickoeff) [inline src/routes/home/BakeOurProject.tsx:30]: If you try to implement toast at promises which have loading, success, error. You can try implement this Promise-Based Toast. Example code:

```
function PromiseBasedToastExample() {
  const toast = useToast()
  return (
    <Button
      onClick={() => {
        // Create an example promise that resolves in 5s
        const examplePromise = new Promise((resolve, reject) => {
          setTimeout(() => resolve(200), 5000)
        })

        // Will display the loading toast until the promise is either resolved
        // or rejected.
        toast.promise(examplePromise, {
          success: { title: 'Promise resolved', description: 'Looks great' },
          error: { title: 'Promise rejected', description: 'Something wrong' },
          loading: { title: 'Promise pending', description: 'Please wait' },
        })
      }}
    >
      Show Toast
    </Button>
  )
}
```

Or if in useAsyncCall have been handled the toast, try to custome that

**Review by vickoeff — CHANGES_REQUESTED** [review]
Commented

File: src/routes/home/BakeOurProject.tsx (hunk 1 above)
Comment (Julius-745) [reply src/routes/home/BakeOurProject.tsx:30]: i have try that before. when i ask you personally at DM i already use it but the success and error toast cant be removed. then i ask you is the wallet toast can be edit or not

Comment (vickoeff) [reply src/routes/home/BakeOurProject.tsx:30]: Ok. But why just only this async call did you add loading toast? If you fix it in useAsyncCall, other async call will be fixed in the same style/behavior. So let our UX have experience consistently

**Review by vickoeff — APPROVED** [review]
LGTM for now. But dont forget to improve quality later

**Pull request merged**